### PR TITLE
[WIP] Update EinsteinPy projects

### PR DESCRIPTION
**Pull request by ritzvik**
@shreyasbapat Please oversee the changes made.

### CODE REVIEW

**Comment by ritzvik** [comment]
I don’t know why the changes are not getting rendered in the CI-built website :(

**Comment by shreyasbapat** [comment]
@Cadair Please don't review as of now. We will complete every other thing I'm this PR itself.

**Comment by ritzvik** [comment]
> @Cadair Please don't review as of now. We will complete every other thing I'm this PR itself.

That’s what I was hoping for.

**Comment by shreyasbapat** [comment]
@Cadair  Please merge! There's nothing to change. Sorry for the delay

File: _projects/2020/einsteinpy/Null-geodesics-for-Schwarzschild-and-Kerr-Geometries..md [hunk 1]
```diff
@@ -37,16 +37,24 @@ collaborating_projects:
...
 
-Further, it would allow us to calculate the ergosphere around a Kerr black hole. As in the trajectory of photon through space-time, the proper time remains zero, we generally have to introduce an affine parameter for the geodesic equations. Equations are generally obtained from analysis of radial and axial momentums of photon as energy and angular momentum are constants of motion in Kerr space-time.
+The project has roughly the following objectives :
+  - Calculating null trajectories around a rotating/non-rotating black hole.
+  - Use those trajectories to raytrace a bundle of photon rays, to visualize how space-time gets distorted in presence of black holes.
+  - Calculate redshifts in photon rays due to space-time curvature.
+  - Visualization of Keplerian disc/shell in around it.
+  - In short, we aim to replicate one of the few papers available with proper coding paradigms like OOPs, good documentation and unit tests.
+  - If required due to perfromance issues, python wrappers over native C/C++ have to be written.
```
Comment (mirca) [inline _projects/2020/einsteinpy/Null-geodesics-for-Schwarzschild-and-Kerr-Geometries..md]:
```suggestion
  - If required due to performance issues, python wrappers over native C/C++ have to be written.
```

**Pull request merged**